x-pack/legacy/plugins/graph/public/components/graph_visualization/_graph_visualization.scss

```diff
@@ -1,3 +1,14 @@
+.gphVisualization {
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+}
+
+.gphGraph {
+  flex: 1;
+  overflow: hidden;
```

Comment:
Set overflow hidden to the graph for IE (other browsers do this by default).
At least this one was also released in the last version (didn't check others):
<img width="665" alt="Screenshot 2019-10-09 at 11 52 16" src="https://user-images.githubusercontent.com/1508364/66471334-6e190d00-ea8b-11e9-9e02-12f1e6b6d082.png">